data/crac/crac-impl/src/test/java/com/farao_community/farao/data/crac_impl/json/CracImportExportTest.java

```diff
@@ -72,46 +72,40 @@ public void cracTest() {
 
         simpleCrac.addNetworkElement(new NetworkElement("pst"));
         simpleCrac.addNetworkElement(new NetworkElement("injection"));
-        simpleCrac.addNetworkAction(new PstSetpoint("pstSetpointId", "pstSetpointName", "RTE", usageRules, simpleCrac.getNetworkElement("pst"), 15, CENTERED_ON_ZERO));
 
-        Set<AbstractElementaryNetworkAction> elementaryNetworkActions = new HashSet<>();
-        PstSetpoint pstSetpoint = new PstSetpoint(
-                "pstSetpointId",
-                "pstSetpointName",
-                "RTE",
-                new ArrayList<>(),
+        // network action with one pst set point
+        PstSetpointImpl pstSetpoint1 = new PstSetpointImpl(
+            simpleCrac.getNetworkElement("pst"),
+            15,
+            CENTERED_ON_ZERO);
+
+        simpleCrac.addNetworkAction(new NetworkActionImpl("pstSetpointRaId", "pstSetpointRaName", "RTE", usageRules, Collections.singleton(pstSetpoint1)));
```

Comment:
You can use Set.of(...) as well to harmonize

Reply:
changed